packages/fulfillment/src/joiner-config.ts

```diff
@@ -17,9 +32,24 @@ Object.entries(LinkableKeys).forEach(([key, value]) => {
 
 export const entityNameToLinkableKeysMap: MapToConfig = entityLinkableKeysMap
 
+const aliases = [
```

Comment:
suggestion(non-blocking): Replace with inline alias definition for readability (and consistency with other modules). This is an unnecessary "optimization"

Reply:
Alterntively, we should apply it to all other modules.

Reply:
Yeah I was trying to propose something before applying everywhere but I ve deleted it to do it like the other modules. The main idea behind was that since the format is known and can be done automatically, it was easier to read for bigger modules. but let see that another time

Reply:
It would be amazing if this was done automatically, as in we don't need either of these approaches.

Reply:
I think it can, I can do a test